.github/workflows/workflow.yml

```diff
@@ -197,3 +209,59 @@ jobs:
       - run: yarn start ../../rococo-local/config.json & cd ../wait-for-inclusion && npm start 200
         name: Launch rococo local & wait for inclusion
         working-directory: scripts/polkadot-launch
+  
+  build-and-publish-docker-image:
+    needs: [pop-runner-instance, build]
+    runs-on: ${{ needs.pop-runner-instance.outputs.runner }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+      - uses: actions/download-artifact@v2
+        name: Download basilisk binary
+        with:
+          name: basilisk
+          path: docker
+      - name: Sleep for 60 seconds
+        run: sleep 60
+      - name: install docker
+        run: sudo snap install docker
+      - name: Sleep for 30 seconds
+        run: sleep 30s
+      - name: build docker image
+        run: sudo docker build -t basilisk:latest docker
+      - name: get ecr credentials
+        run: aws ecr get-login-password --region eu-west-1 | sudo docker login --username AWS --password-stdin 601305236792.dkr.ecr.eu-west-1.amazonaws.com
+      - name: push image to ecr repository
+        run: sudo docker tag basilisk:latest 601305236792.dkr.ecr.eu-west-1.amazonaws.com/basilisk
+      - run: sudo docker push 601305236792.dkr.ecr.eu-west-1.amazonaws.com/basilisk:latest
+  
+  inform-build-success:
+    needs: test-inclusion
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-ruby@v1
+      - name: Send Webhook Notification
+        if: always()
+        env:
+          JOB_STATUS: ${{ job.status }}
+          WEBHOOK_URL: ${{ secrets.DISCORD_WEBHOOK_URL }}
+          HOOK_OS_NAME: ${{ runner.os }}
+          WORKFLOW_NAME: ${{ github.workflow }}
+        run: |
```

Comment:
can we get a notification on successful deployment as well, not only build?

Reply:
Yup. Adding that. Will resolve conversation once done

Reply:
Added in the testnet-deployment repo